.github/workflows/gradle_build_and_test.yml

```
@@ -40,3 +37,38 @@ jobs:
+          push: true
+          tags: |
+            ${{ secrets.DOCKERHUB_USERNAME }}/cherry-be-dev:latest
+            ${{ secrets.DOCKERHUB_USERNAME }}/cherry-be-dev:${{ github.sha }}
+          build-args: |
+            JWT_SECRET=${{ secrets.JWT_SECRET }}
+            ACCESS_TOKEN_EXPIRATION_SECONDS=${{ secrets.ACCESS_TOKEN_EXPIRATION_SECONDS }}
+            REFRESH_TOKEN_EXPIRATION_SECONDS=${{ secrets.REFRESH_TOKEN_EXPIRATION_SECONDS }}
+            KAKAO_CLIENT_ID=${{ secrets.KAKAO_CLIENT_ID }}
+            KAKAO_REDIRECT_URL=${{ secrets.KAKAO_REDIRECT_URL }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)